app/templates/dayview.html

```diff
@@ -10,38 +10,49 @@
-            <a href="/closethedayview" title="close day view"><img class="pb-1" src="{{ url_for('static', path='/images/icons/close_sidebar.svg')}}" width="15em" height="15em"></a>
-            <span class="month fw-bold text-white">{{month}}</span>
-            <span class="day fw-bold text-white">{{day}}</span>
-            {% if zodiac %}
-            <div class="zodiac-sign">
-                <a href="https://www.horoscope.com/zodiac-signs/{{zodiac.name}}" title="Zodiac-sign: {{zodiac.name}}" target="_blank"><img src="{{ url_for('static', path='/images/zodiac/' + zodiac.name + '.svg') }}" alt="zodiac sign" width="14em" height="13em"></a>
-            </div>
+            {% if view == 'day' %}
+                <a href="/closethedayview" title="close day view"><img class="pb-1" src="{{ url_for('static', path='/images/icons/close_sidebar.svg')}}" width="15em" height="15em"></a>
+                <span id="month-num" class="month-num fw-bold text-white">{{month}}</span>
+                <span id="day-num" class="day-num fw-bold text-white">{{day}}</span>
+                {% if zodiac %}
+                  <div class="zodiac-sign">
+                      <a href="https://www.horoscope.com/zodiac-signs/{{zodiac.name}}" title="Zodiac-sign: {{zodiac.name}}" target="_blank"><img src="{{ url_for('static', path='/images/zodiac/' + zodiac.name + '.svg') }}" alt="zodiac sign" width="14em" height="13em"></a>
+                  </div>
+                {% endif %}
+            {% else %}
+                <span class="fw-bold text-white date-nums">{{day}} / {{month}}</span>
             {% endif %}
         </div>
         <div class="schedule">
             <div class="container times bg-primeary position">
-                {% for i in range(24)%}
-                    <div class="row bg-transparent hourmark">
-                    {% set i = i|string() %}
-                    {{i.zfill(2)}}:00
-                    </div>
-                {% endfor %}
+                {% if view == 'day'%}
+                    {% for i in range(24)%}
+                        <div class="row bg-transparent hourmark">
+                        {% set i = i|string() %}
+                        {{i.zfill(2)}}:00
+                        </div>
+                    {% endfor %}
+                {% endif %}
             </div>
             <div class="eventgrid">
                 {% for event, attr in events %}
                     {% set totaltime = 'visible'%}
-                    {% if attr.length < 60 %}
-                            {% set size = 'title_size_small' %}
-                            {% set totaltime = 'invisible'%}
-                        {% if attr.length < 45 %}
-                                {% set size = 'title_size_Xsmall' %}
-                            {% if attr.length < 30 %}
-                                    {% set size = 'title_size_tiny' %}
+                    {% if attr.length <= 90 %}
```

Comment:
Use [`bisect`](https://docs.python.org/3/library/bisect.html) inside Python and just send the final size to make it a bit more elegant :)